src/APICore.ts

```diff
@@ -12,23 +12,36 @@ import {handleRequest} from './handlers/request/RequestHandlers.js';
 
 interface TokenInfo {
     authentication: UserModel;
-    [key: string]: any;
+    [key: string]: unknown;
 }
 
 const HEADERS_JSON_CONTENT_TYPE: HeadersInit = Object.freeze({'Content-Type': 'application/json'});
 
-/** Check whether the response status is `429 Too Many Requests`. */
+/**
+ * Check whether the response status is `429 Too Many Requests`.
+ * @param response
+ */
 const isTooManyRequests: Predicate<Response> = (response) => response.status === 429;
-/** Check whether the method is 'GET' (case insensitive). */
-const isGet: Predicate<string | undefined> = RegExp.prototype.test.bind(/^GET$/i);
-
-/** "Logical OR" two optional abort signals. */
+/**
+ * Check whether the method is 'GET' (case insensitive).
+ * @param value
+ */
+const isGet: Predicate<string | undefined> = (value) => (value ? /^GET$/i.test(value) : false);
+
+/**
+ * "Logical OR" two optional abort signals.
+ * @param signal1
+ * @param signal2
+ */
 const abortOnEither = (
     signal1: AbortSignal | null | undefined,
     signal2: AbortSignal | null | undefined,
 ): AbortSignal | null | undefined => {
     if (signal1 && signal2) {
         const joined = new AbortController();
+        /**
+         *
+         */
```

Comment:
Is this needed ?

Reply:
I think it was added automatically by the eslint-plugin-jsdocs 🤔 It looks like it wants to force documenting everything that is publicly reachable from the user of the class

Reply:
I will leave it be for now since it does not harm anything. We can always refine the lint rules afterwards.